code/game/objects/items/oddities.dm

```diff
@@ -390,6 +389,8 @@
 	name = "Strange Device"
 	desc = "You can't find out how to turn it on. Maybe it's already working?"
 	icon_state = "artwork_1"
+	price_tag = 200
```

Comment:
Why have all artwork oddities only be worth 200$?

Reply:
the export price increases with respect to its stats.

Reply:
You can just code where on initialization it gives odditie a price based on its stats.

Reply:
The variable price tag should not be modified except for special circumstances, that is why I have the stats change the export price.

Reply:
> You can just code where on initialization it gives odditie a price based on its stats.
Besides this oddity starts without stats.